src/services/completions.ts

```diff
@@ -1640,7 +1640,8 @@ namespace ts.Completions {
             completionKind = CompletionKind.MemberLike;
             // Declaring new property/method/accessor
             isNewIdentifierLocation = true;
-            keywordFilters = isClassLike(decl) ? KeywordCompletionFilters.ClassElementKeywords : KeywordCompletionFilters.InterfaceElementKeywords;
+            keywordFilters = contextToken.kind === SyntaxKind.AsteriskToken ? KeywordCompletionFilters.None :

```

Comment:
Why is this none? If its in class/method you want to add no keywords at all and otherwise(for function) you want to have KeywordCompletionFilters.None?

Reply:
Apparently we're using `KeywordCompletionFilters.None` to alternately mean "None" or "All" depending on whether it's a member completion. :man_facepalming: